src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -200,6 +206,8 @@ def get_obs_data(bufr, profile_meta_data, add_qc, addLSW, record_number=None):
     impact = codes_get_array(bufr, 'impactParameter')[offset::drepfac[0]]
     bang = codes_get_array(bufr, 'bendingAngle')[offset*2::drepfac[0]*2]
     bang_err = codes_get_array(bufr, 'bendingAngle')[offset*2+1::drepfac[0]*2]
+    if only_bang:
+        bang_err[:] = 0.003
```

Comment:
I tested the decoded data and found that missing bang_error doesn't cause an issue while running H(x). So this line seems not necessary.

Reply:
we need to open an issue an can remove this line